apps/server/src/api-integration/integration.router.ts

```diff
@@ -36,14 +36,14 @@ integrationRouter.get('/*', (req: Request, res: Response) => {
 
   try {
     const actionArray = action.split('/');
-    const params = { payload: req.query as object } as { payload: object | null };
-
+    const params = req.query as object;
+    let payload = params;
```

Comment:
Would this not simply copy the reference to a new object? If we are trying to dissociate payload so it can be mutated in this context, we likely need to spread params 

We should also likely be more defensive since we don't know for sure what is inside params